package.json

```diff
@@ -2,6 +2,13 @@
   "name": "tutanota",
   "version": "3.25.11",
   "scripts": {
+    "pretest": "npm run-script flow",
+    "test": "npm run-script testapi && npm run-script testclient",
+    "pretestapi": "mkdir -p build/src/api/worker && test/api.sh server spawn \"$(pwd)/server.js\"",
+    "testapi": "( cd test && node test.js api; )",
```

Comment:
I see something similar to this included in [merged code](https://github.com/tutao/tutanota/commit/0708dae2#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12), though not sure how that code handles:
- Potential bad interactions with other `server.js` instances.
- Distinct exit codes.

(See test/api.sh in this PR for details.)